jupyterlab_git/git.py

```diff
@@ -1540,3 +1595,142 @@ async def tag_checkout(self, path, tag):
                 "command": " ".join(command),
                 "message": error,
             }
+
+    async def check_credential_helper(self, path: str) -> Dict[str, Any]:
```

Comment:
as we do not intent to report this back to the frontend - please return straigh type or raise an error.

in this case I would return True, False or None
True => has helper and needs cache daemon
False => has helper and does not need cache daemon
None => No helper